docs/datamap/mv-datamap-guide.md

```diff
@@ -65,6 +65,7 @@ EXPLAIN SELECT a, sum(b) from maintable group by a;
   CREATE DATAMAP agg_sales
   ON TABLE sales
   USING "MV"
+  DMPROPERTIES('TABLE_BLOCKSIZE'='256 MB','LOCAL_DICTIONARY_ENABLE'='false')
```

Comment:
Why this is needed in this doc?

Reply:
Usage of DMProperties has been added in the example,  so that user can check and provide provide tableproperties and datamap properties also for child datamap